common/views/components/Header/HeaderSearch.tsx

```diff
@@ -0,0 +1,83 @@
+import { Dispatch, SetStateAction, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { useRouter } from 'next/router';
+import { classNames } from '@weco/common/utils/classnames';
+import SearchBar from '@weco/common/views/components/SearchBar/SearchBar';
+import Space from '@weco/common/views/components/styled/Space';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import { arrowSmall } from '@weco/common/icons';
+
+const Overlay = styled.div.attrs<{ isActive: boolean }>(props => ({
+  className: classNames({
+    'is-hidden': !props.isActive,
+  }),
+}))<{ isActive: boolean }>`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
+  z-index: 9999;
+`;
+
+const SearchBarWrapper = styled(Space).attrs({
+  v: { size: 'xl', properties: ['padding-bottom'] },
+})`
+  position: absolute;
+  width: 100%;
+  ${props => props.theme.makeSpacePropertyValues('l', ['padding-top'])};
+  background-color: ${props => props.theme.color('white')};
+`;
+
+type Props = {
+  isActive: boolean;
+  setIsActive: Dispatch<SetStateAction<boolean>>;
+};
+
+const HeaderSearch = ({ isActive, setIsActive }: Props) => {
+  const nodeRef = useRef(null);
```

Comment:
What is `nodeRef` used for? I can see it being set but not being used anywhere

Reply:
Honestly think this is cleanup worthy - tried to do something to target the wrapper and can't even remember what, thank you